docs/libraries/nodejs/examples.md

```diff
@@ -1,101 +1,149 @@
+
+The most common methods of `account` instance:
+* `account.alias()`: returns an alias of the given account
+* `account.listAddresses()`: returns list of addresses related to the account
+* `account.getUnusedAddress()`: returns a first unused address
+* `account.generateAddress()`: generate a new address for the address index incremented by 1
+* `account.balance()`: returns the balance for the given account
+* `account.sync()`: sync the account information with the tangle
+
+## Generating address(es)
+Each account can posses multiple addresses. Addresses are generated deterministically based on the account and address index. It means that the combination of account and index uniquely identifies the given address.
+
+Addresses are of two types: `internal` and `public` (external):
+* each set of addresses is independent from each other and has independent `index` id
+* addresses that are created by `account.generateAddress()` are indicated as `internal=false` (public)
+* internal addresses (`internal=true`) are so called `change` addresses and are used to send the excess funds to
+* the approach is also known as a *BIP32 Hierarchical Deterministic wallet (HD Wallet)*.
+
+_Note: You may remember IOTA 1.0 network in which addresses were not reusable. It is no longer true and addresses can be reused multiple times in IOTA 1.5 (Chrysalis) network._
 
-Code:
 ```javascript
 {{ #include ../../../bindings/nodejs/examples/2-generate-address.js }}
 ```
 
-## 3. Example: Check Balance
-Run:
-```
-node 3-check_balance
-```
+## Checking balance
+Before we continue further, go to [IOTA testnet faucet service](https://faucet.testnet.chrysalis2.com/) and send to your testnet addresses some tokens.
 
-Code:
 ```javascript
 {{ #include ../../../bindings/nodejs/examples/3-check_balance.js }}
 ```
 
-## 4. Example: Check Balance
-Now you can send the test tokens to an address! 
-
-Run
-```
-node 4-send.js
-```
+## Sending tokens
+Sending tokens is performed via `SyncedAccount` instance that is a results of `account.sync()` function:
 
-Code:
 ```javascript
 {{ #include ../../../bindings/nodejs/examples/4-send.js }}
 ```
+The full function signature is `SyncedAccount.send(address, amount[, options])`.
+Default options are perfectly fine and do the job done, however additional options can be provided, such as `remainderValueStrategy`:
+* `changeAddress`: Send the remainder value to an internal address
+* `reuseAddress`: Send the remainder value back to its original address
 
-## 5. Backup
+`SyncedAccount.send()` function returns a `wallet message` that fully describes the given transaction. Especially `messageId` can later be used for checking a confirmation status. Individual messages related to the given account can be obtained via `account.listMessages()` function.
```

Comment:
SyncedAccount is no longer a thing unfortunately :( the `send` method is not part of the Account object.

Reply:
Fixed. Will make sure it is also fixed on Exchange Guide. It was not an issue in Python docs BTW. https://github.com/iotaledger/wallet.rs/pull/437/commits/ef21c35cf2fb1943ffb7023cb18ed8f7fbb9863a